src/js/components/stock-movement-wizard/request/CreateStockMovement.jsx

```diff
@@ -278,7 +278,7 @@ class CreateStockMovement extends Component {
    */
   fetchStockLists(origin, destination, clearStocklist) {
     this.props.showSpinner();
-    const url = `/api/stocklists?origin.id=${origin.id}&destination.id=${destination.id}`;
+    const url = `/api/stocklists?origin=${origin.id}&destination=${destination.id}`;
```

Comment:
I'm nervous about removing the ".id" part of the parameter name without understanding what the underlying issue is. It's one thing if this was an isolated issue, but it feels like this is going to be the start of a very uncomfortable refactoring. 

If you have further information on why this is necessary, please add your investigation to the ticket. Otherwise we're going to have to turn this issue into a spike to figure out the extent of the problem.

Reply:
@jmiranda I would not be nervous, because it seems like this issue was once solved on Grails 1 as well, but just got lost somewhere while rebasing.
Currently on Grails 1 we don't have `origin.id` being used, but `origin`, like:
```javascript
const url = `/openboxes/api/stocklists?origin=${origin.id}&destination=${destination.id}`;
```

Commit where it was fixed: https://github.com/openboxes/openboxes/commit/65a3859f349624a13ab96cfc1db158595f8712ca
and the PR: https://github.com/openboxes/openboxes/pull/3498
Feels like @awalkowiak was the one fixing it on Grails 1

cc @alannadolny

Reply:
@kchelstowski I changed the hard-coded URL as u proposed.